.github/CONTRIBUTING.md

```diff
@@ -186,32 +186,31 @@ At any rate, let's have some fun!
 ### Articles
 
 As a bonus to this repository, there are associated articles for every script.
-However, the articles do take awhile to write, so you can help by writing them
+However, the articles do take a while to write, so you can help by writing them
 as well.
 
 Previously, we had hosted all of the articles on [The Renegade Coder][34], but
 we've since moved away from that to support open-source editing. Now, you
 can add and update any article you want.
 
-Currently, there are three types of articles you can write: code, project, and
+Currently, there are three types of articles you can write: code, project and
```

Comment:
I actually prefer the Oxford comma, but it doesn't really matter.

Reply:
Accepting this, ..